x-pack/test/functional/services/ml/settings.ts

```diff
@@ -10,12 +10,20 @@ export function MachineLearningSettingsProvider({ getService }: FtrProviderConte
   const testSubjects = getService('testSubjects');
 
   return {
-    async assertSettingsCalendarLinkExists() {
-      await testSubjects.existOrFail('ml_calendar_mng_button');
+    async assertSettingsManageCalendarsLinkExists() {
+      await testSubjects.existOrFail('ml_calendars_mng_button');
```

Comment:
We mostly use camelCase for test subjects. The settings subjects are historically grown snake_case (they were taken over that way from test ids). Since you're already editing them, we could think about making the case consistent with other parts of the code.